frontend/packages/client/src/components/CommunityProposals.js

```diff
@@ -54,10 +61,12 @@ export default function CommunityProposals({ community = { id: 1 } }) {
             extraClasses="is-5"
             extraClassesMobile="mt-2 mb-5 is-8 pr-2"
           >
-            <DropDownFilter
-              value={filterValue}
-              filterValues={proposalFilterValues}
-              setFilterValues={setFilterValues}
+            <DropDown
+              defaultValue={{ value: "all", label: "All" }}
+              values={proposalFilterValues}
+              onSelectValue={(value) => {
+                setFilterValues(value);
+              }}
```

Comment:
I realized that we have the component Dropdown that should replace  `DropDownFilter` component. Once we merge this feature branch for community-v2 DropDownFilter could be safely deleted. I did the changes to use Dropdown now so we don't create more dependencies to the old component.